code/modules/projectiles/guns/energy/special.dm

```diff
@@ -79,35 +79,37 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/mindflayer)
 	ammo_x_offset = 2
 
-/obj/item/gun/energy/kinetic_accelerator/crossbow
+/obj/item/gun/energy/crossbow
 	name = "mini energy crossbow"
 	desc = "A weapon favored by syndicate stealth specialists."
 	icon_state = "crossbow"
 	item_state = "crossbow"
+	force = 4
+	damtype = TOX
+	attack_verb = list("poisoned", "jabbed")
 	w_class = WEIGHT_CLASS_SMALL
-	materials = list(MAT_METAL=2000)
+	cell_type = /obj/item/stock_parts/cell{charge = 1000; maxcharge = 1000}

```

Comment:
I'm pretty sure there's a cell that already has these numbers. And again, this is gross. IF there's not, make a subtype and use it here, don't use curly brackets to set vars on a type.

Reply:
my excuse is the mini-egun does it so I went off that.